### ISSUE TITLE
Waveform from a Stream<Uint8List>

### ISSUE THREAD

**Issue by TarekMadara** [ryanheise/just_waveform#20]
I have a Stream<Uint8List> filled from the Network, is there a way to get the waveform from this ?

**Comment by ryanheise** [comment]
Yes, write the data to a file, then you can pass the file into this API.
